composer.json

```diff
@@ -62,7 +62,10 @@
     "autoload-dev": {
         "psr-0": {
             "": "engine/tests/phpunit/"
-        }
+        },
+		"files": [
+			"engine/tests/phpunit/bootstrap.php"
```

Comment:
I see you sniff for a constant and bail if not testing. Still, autoloading this file feels dirty.